lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -47,6 +47,34 @@
   }
 }
 
+
+.nx-multi-file-upload {
+  width: 348px;
+
+  // need an enclosing div around NxMultiFileUpload that doesn't contain the validationError message
+  &__container {
+    $height: 220px;
+
+    min-height: $height;
+    box-sizing:border-box;
```

Comment:
```suggestion
    box-sizing: border-box;
```

Reply:
spacing fixed here (in NxMultiFileUpload.scss) : [dcfb41a](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/dcfb41af21eceb71843ee6146ba1b79049e60aa0)